arroyo/processing/strategies/produce.py

```diff
@@ -53,10 +63,14 @@ def poll(self) -> None:
         while self.__queue:
             committable, future = self.__queue[0]
 
-            if not future.done():
+            if future is not None and not future.done():
                 break
 
-            message = Message(Value(future.result().payload, committable))
+            message: Message[TPayload]
+            if future is None:
+                message = Message(Value(FILTERED_PAYLOAD, committable))
```

Comment:
I didn't find a way to create a future from scratch that resolves immediately. Even creating an `async def` doesn't seem to work since it returns a coroutine instead.

Reply:
can you do something like:
````
fut = asyncio.Future()
fut.set_result(FILTERED_PAYLOAD)
````